Content.Tests/DMProject/Tests/Expression/Derefs/new_deref.dm

```diff
@@ -0,0 +1,16 @@
+
+//# issue 1276
+
+#define ASSERT(x) x
```

Comment:
Why do you disable `ASSERT()` here?

Reply:
Ah, I was supposed to remove that. I was debugging the compiler, but since it loaded the standard library which would lead me into stacks that I was absolutely not interested in debugging. Since ASSERT is stdlib, I had to define it so I could debug the compiler on this particular test case. I was supposed to remove this `#define` before PR-ing it, which I did neglect to in another PR until I caught and fixed it myself.